Dockerfile

```
@@ -8,6 +8,7 @@ COPY package.json yarn.lock /app/
 RUN yarn
 
 COPY . /app
+RUN yarn cs
```

```suggestions
RUN yarn cs && yarn build
```

this will reduce number of layers in the docker image. not critical here but for more advance setups that is docker recommended practice

---

👍